tensorflow_serving/tools/docker/Dockerfile.gpu

```
@@ -49,7 +49,8 @@ RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/
         libfreetype6-dev \
         pkg-config \
         software-properties-common \
-        unzip
+        unzip \
+        cuda-toolkit-11-8
```

```suggestion
        cuda-compiler-11-8
```
Though installing the entire `cuda-toolkit` solves the issue in #2201, it increases the image size by up to 4GB. I have tried alternatives and it seems sufficient to install the [`cuda-compiler` metapackage](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/#meta-packages).